addons/mail_group/tests/common.py

```diff
@@ -30,6 +30,13 @@ def setUpClass(cls):
             'moderator_ids': [Command.link(cls.user_employee.id)],
             'name': 'Test group',
         })
+        cls.test_group_2 = cls.env['mail.group'].create({
```

Comment:
Seems used only in some tests, would not create it by default (helps understanding tests and data in my opinion).